src/cloudai/workloads/nemo_run/slurm_command_gen_strategy.py

```diff
@@ -71,6 +84,25 @@ def append_flattened_dict(self, prefix: str, d: Dict[str, Any], command: List[st
                 else:
                     command.append(f"{key}={value}")
 
+    def _validate_recipe_name(self, recipe_name: str) -> str:
+        """Validate the recipe name against the supported list."""
+        supported_recipes = [
+            "cloudai_llama3_8b_recipe",
+            "cloudai_llama3_70b_recipe",
+            "cloudai_llama3_405b_recipe",
+            "cloudai_nemotron3_8b_recipe",
+            "cloudai_nemotron4_15b_recipe",
+            "cloudai_nemotron4_340b_recipe",
+        ]
+
+        if recipe_name not in supported_recipes:
+            logging.warning(
+                f"Using default {recipe_name} in Nemo2.0."
```

Comment:
Need a space after `Nemo2.0.`, otherwise warning will look like this `... in Nemo2.0.Passing advance ...`